starter-code/bin/seeds.js

```diff
@@ -1 +1,37 @@
+'use strict';
 // Iteration #1
+
+
+const Drone = require("../models/drone").Drone;
+
+const mongoose = require("mongoose");
+
+mongoose.connect('mongodb://localhost:27017/drones');
+
+const droneData = [{
+          droneName: "R2D2",
+          propellers: 4,
+          maxSpeed: 20,
+     },
+     {
+          droneName: "C3PO",
+          propellers: 8,
+          maxSpeed: 25,
+     },
+     {
+          droneName: "BB8",
+          propellers: 4,
+          maxSpeed: 15,
+     },
+];
+
+Drone.create(droneData, (err, res) => {
+     if (err) {
+          console.log(err);
+     } else {
+          console.log(res);
+     }
+});
```

Comment:
Here is where you want to close the connection with the database, in the callback of the create method. That way, you are sure the connection is opened until all the documents are created. The instruction is:
```
mongoose.disconnect();
```